pkg/services/sqlstore/migrations/ualert/migration_service_migration.go

```diff
@@ -0,0 +1,69 @@
+package ualert
+
+import (
+	"fmt"
+	"os"
+	"strconv"
+	"time"
+
+	"xorm.io/xorm"
+
+	"github.com/grafana/grafana/pkg/services/sqlstore/migrator"
+)
+
+// KVNamespace is a vendored migration.KVNamespace.
+var KVNamespace = "ngalert.migration"
+
+// migratedKey is a vendored migration.migratedKey.
+var migratedKey = "migrated"
+
+// MigrationServiceMigration moves the legacy alert migration status from the migration log to kvstore.
+func MigrationServiceMigration(mg *migrator.Migrator) {
```

Comment:
This is necessary so that we retain the current migration state between when sqlstore was in charge of UA migration and now. See 79781aefcc995be95c2dff8b02099aff39abf8af